azure_arc_servers_jumpstart/docs/gcp_terraform.md

```diff
@@ -0,0 +1,117 @@
+
+# Deployment
+
+Before executing the Terraform plan, you must export the environment variables which will be used by the plan. These variables are based on the Azure Service Principal you've just created, your Azure subscription and tenant, and the GCP project name.
+
+* Retrieve your Azure Subscription ID and tenant ID using the ```az account list``` command.
+
+* The Terraform plan creates resources in both Microsoft Azure and Google Cloud. It then executes a script on a Google Cloud virtual machine to install the Azure Arc agent and all necessary artifacts. This script requires certain information about your Google Cloud and Azure environments. Edit ***scripts/vars.sh*** and update each of the variables with the appropriate values.
+    ![](../img/gcp/06.png)
+
+* Open Az CLI and navigate to the ```azure_arc_servers_jumpstart/gcp/terraform``` directory of the cloned repo.
+
+* Export the environment variables you edited by running the vars.sh script with the source command as shown below.
+
+    ```source ./scripts/vars.sh```
+
+* Run the ```terraform init``` command which will download the Terraform AzureRM provider.
+
+    ![](../img/gcp/08.png)
+
+* Next, run the ```terraform apply --auto-approve``` command and wait for the plan to finish. Once finished, you will be able to manage the Google Cloud virtual machine from Azure.
+
+# View the GCP VM from Azure Portal
+
+* Open the Azure portal and navigate to the resource group "Arc-Servers-Demo". The virtual machine created in GCP will be visible as a resource.
+
+    ![](../img/gcp/09.png)
+
+* stuff about managing resource
+
+# Semi-Automated Deployment (Optional)
+
+As you may noticed, the last step of the run is to register the VM as a new Arc server resource. 
+
+    ![](../img/gcp/10.png)
+
+If you want to demo/control the actual registration process, do the following: 
+
+1. In the [*deploy_arcagent.sh.tmpl*](../gcp/terraform/scripts/deploy_arcagent.sh.tmpl) script template, comment out the "Run connect command" section and save the file.
+
+    ![](../img/gcp/11.png)
+
+2. Get the public IP of the GCP VM by running ```terraform output```
+
+    ![](../img/gcp/12.png)
+
+3. SSH the VM using the ```ssh arcadmin@x.x.x.x``` where x.x.x.x is the host ip. 
+
+    ![](../img/gcp/13.png)
+
+4. Export all the environment variables in [*vars.sh*](../gcp/terraform/scripts/vars.sh)
+
+    ![](../img/gcp/14.png)
+
+5. Run the following command
+    ```azcmagent connect --service-principal-id $TF_VAR_client_id --service-principal-secret $TF_VAR_client_secret --resource-group "Arc-Servers-Demo" --tenant-id $TF_VAR_tenant_id --location "westus2" --subscription-id $TF_VAR_subscription_id```
+
+    ![](../img/gcp/15.png)
+
+6. When complete, your vm will be registered with Azure Arc and visible in the resource group inside Azure Portal.
+
+# Delete the deployment
+
+To delete all the resources you created as part of this demo use the ```terraform destroy --auto-approve``` command as shown below.
```

Comment:
Let's also show how to nuke from the GCP console

Reply:
added this